tools/squashfs4/Makefile

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2006-2012 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=squashfs4
+PKG_CPE_ID:=cpe:/a:phillip_lougher:squashfs
+PKG_VERSION:=4.5.1
+PKG_RELEASE=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/plougher/squashfs-tools
+PKG_SOURCE_DATE:=2022-17-03
```

Comment:
This date format is wrong. I prefer `2022-03-17`.

Reply:
It should be `2022-03-18`.

Reply:
Shouldn't it be
```
PKG_SOURCE_DATE:=2023-02-08
PKG_SOURCE_VERSION:=0f5666a3b250d7732211c188b58d11b36744c75e
```

Which is the current `HEAD` over there, and the date of the commit.

Reply:
we are targeting the release hash.
@lovehackintosh why 18 ?

Reply:
Sorry, I got it wrong, github showed the date as 2022-03-18, but the real commit date was 2022-03-17.
![IMG_20230214_125852](https://user-images.githubusercontent.com/92633080/218643612-7a239c38-4d34-406a-98aa-b72f57c61914.jpg)
![IMG_20230214_125943](https://user-images.githubusercontent.com/92633080/218643631-7850dab8-d613-4524-a629-f82783298d6b.jpg)